Difference in MATLAB Code block 'Add One'
CHANGED block:
==============================================================================
--- ./test1.slx

+++ ./test1.slx_fixed

@@ -1,3 +1,3 @@

 function y = add_one(u)
 
-y = u + 1;
+    y = u + 1;
==============================================================================

Difference in MATLAB Code block 'Multiply'
CHANGED block:
==============================================================================
--- ./test1.slx

+++ ./test1.slx_fixed

@@ -1,3 +1,3 @@

 function y = my_multiply(u, v)
 
-y = u * v;
+    y = u * v;
==============================================================================

Difference in MATLAB Code block 'Sub One'
CHANGED block:
==============================================================================
--- ./test1.slx

+++ ./test1.slx_fixed

@@ -1,3 +1,3 @@

 function y = sub_one(u)
 
-y = u - 1;
+    y = u - 1;
==============================================================================
